### PR TITLE
User.com doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Created User.com doc

### CODE REVIEW

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+    );
+```
+
+### Property mapping
+
+The following table lists the mappings between RudderStack and User.com properties for `identify` call:
+
+| RudderStack property | User.com property | Presence | 
+| :-------| :--------| :------| 
+| `userId` | `custom_id` | Required  |
```
Comment (mihir-4116) [inline docs/destinations/streaming-destinations/user-com.mdx:113]: can we updated all values of a key like traits.userId, context.traits.userId (updated in notion all possible value of Rudderstack property key)

**Review by mihir-4116 — CHANGES_REQUESTED** [review]

File: docs/destinations/streaming-destinations/user-com.mdx (hunk 1 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/user-com.mdx]: done

**Review by mihir-4116 — APPROVED** [review]

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+<img src="../../assets/event-stream-destinations/user-com-connection-settings-3.png" alt="User.com connection settings" />
+
+- **Public REST API Key**: Enter your public User.com REST API key. 
+- **App Subdomain**: Enter the subdomain of your User.com app.
+
+<div class="infoBlock">
+For more information on obtaining the public REST API key and app subdomain, refer to the <Link to="#faq">FAQ</Link> section below.
+</div>
+
+- **Map Rudder user attributes to User.com attributes**: Enter the custom user attributes you want to map for RudderStack and User.com correspondingly.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:43]:
```suggestion
- **Map Rudder user attributes to User.com attributes**: With this setting, you can map the RudderStack event properties to specific User.com user attributes.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+- **Map RudderStack event name to User.com event name**: Enter the event names you want to map for RudderStack and User.com correspondingly. You can map one or more RudderStack events to a single User.com event but not vice-versa.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:44]:
```suggestion
- **Map RudderStack event name to User.com event name**: Use this setting to map the RudderStack event names to specific User.com events. You can map one or more RudderStack events to a single User.com event but **not** vice-versa.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+- **Map your event property**: Enter the RudderStack and User.com event properties you want to map for the above-mentioned event names. You can map a RudderStack property to a single User.com property and vice-versa.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:45]:
```suggestion
- **Map your event property**: Enter the RudderStack and User.com event properties you want to map for the above-mentioned event names. You can map a RudderStack property only to one User.com property and vice-versa.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+- **Map Rudder company attributes to User.com company attributes**: Enter the custom company attributes you want to map for RudderStack and User.com correspondingly.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:46]:
```suggestion
- **Map Rudder company attributes to User.com company attributes**: Use this setting to map the RudderStack company attributes to specific User.com company attributes.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+
+## User lookup
+
+<div class="infoBlock">
+This algorithm to look up a user is applicable only for the <code class="inline-code">identify</code>, <code class="inline-code">track</code>, and <code class="inline-code">page</code> calls.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:51]:
```suggestion
The algorithm to look up a user is applicable only for the <code class="inline-code">identify</code>, <code class="inline-code">track</code>, and <code class="inline-code">page</code> calls.
```

Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:51]:
```suggestion
This functionality is applicable only for the <code class="inline-code">identify</code>, <code class="inline-code">track</code>, and <code class="inline-code">page</code> calls.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+</div>
+
+RudderStack looks up for a user using the `user_key`, `email` or `phone` properties.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:54]:
```suggestion
RudderStack looks up a user in User.com using the `user_key`, `email`, or `phone` properties.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+
+The precedence order for looking up a user based on the above properties is:
+
+- The `userKey` is given the highest priority.
+- The `lookup` value is given the second highest priority. If `phone`/`email` (whichever is passed) is not found, RudderStack will throw an error.
+- If none of the above is present, RudderStack falls back to the `userId`. If not found, RudderStack will throw an error.
+
+## Identify 
+
+You can use the <Link to="/event-spec/standard-events/identify">`identify`</Link> call to create a new user in User.com. If the user already exists, the user details are updated.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:85]:
```suggestion
You can use the <Link to="/event-spec/standard-events/identify">`identify`</Link> call to create a new user in User.com. If the user already exists, RudderStack updates the user details.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+| `traits.googleUrl`/`context.traits.googleUrl` | `google_url` | Optional | 
+| `traits.linkedinUrl`/`context.traits.linkedinUrl` | `linkedin_url` | Optional | 
+| `traits.twitterUrl`/`context.traits.twitterUrl` | `twitter_url` | Optional | 
+| `traits.facebookUrl`/`context.traits.facebookUrl` | `facebook_url` | Optional | 
+| `traits.avatar`/`context.traits.avatar`/`traits.avatarURL`/`context.traits.avatarURL`/`traits.avatar_URL`/`context.traits.avatar_URL` | `gravatar_url` | Optional | 
+| `traits.timezone`/`context.traits.timezone` | `timezone` | Optional | 
+
+## Track
+
+You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to record a new event, which in turn, can be used to filter and bucket users in User.com.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:133]:
```suggestion
You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to record a new event which, in turn, can be used to filter and bucket users in User.com.
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+    user: {
+      lookup: "email"
+    }
+  }
+););
+```
+
+### Property mapping
+
+The following table lists the mappings between RudderStack and User.com properties for `track` call:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:153]:
```suggestion
The following table lists the mappings between RudderStack and User.com properties for the `track` call:
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 11]
```diff
@@ -0,0 +1,255 @@
...
+    city: "New Orleans",
+    region: "Louisiana",
+    country: "USA"
+  },
+);
+```
+
+### Property mapping
+
+The following table lists the mappings between RudderStack and User.com properties for `group` call:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:218]:
```suggestion
The following table lists the mappings between RudderStack and User.com properties for the `group` call:
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 12]
```diff
@@ -0,0 +1,255 @@
...
+| `traits.size` | `size` | Optional | 
+| `traits.tags` | `tags` | Optional | 
+
+## FAQ
+
+### Where can I find the User.com public REST API key?
+
+To get your public REST API key, follow these steps:
+
+1. Log into <a href="https://app.user.com/accounts/login/">User.com dashboard</a> and go to your App.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:242]:
```suggestion
1. Log into your <a href="https://app.user.com/accounts/login/">User.com dashboard</a> and go to your app. 
```

File: docs/destinations/streaming-destinations/user-com.mdx [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+2. Click **Settings** > **App settings** > **Advanced** > **Public REST API keys**.
+3. Click **Create API key**, as shown:
+
+<img src="../../assets/event-stream-destinations/user-com-dashboard.png" alt="User.com connection settings" />
+
+### Where can I find the User.com app subdomain?
+
+To get your app subdomain, follow these steps:
+
+1. Log into <a href="https://app.user.com/accounts/login/">User.com dashboard</a> and go to your App.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/user-com.mdx:252]:
```suggestion
1. Log into your <a href="https://app.user.com/accounts/login/">User.com dashboard</a> and go to your app. 
```

**Review by ameypv-rudder — APPROVED** [review]
LGTM

**Pull request merged**